x-pack/plugins/actions/server/lib/mustache_lambdas.ts

```diff
@@ -46,9 +46,37 @@ function getLambdas(logger: Logger) {
         const numberString = render(text.trim()).trim();
         return formatNumber(logger, numberString);
       },
+    EncodeURI: () =>
+      function (text: string, render: RenderFn) {
+        const string = render(text.trim()).trim();
+        return callEncodeURI(logger, string);
+      },
+    EncodeURIComponent: () =>
+      function (text: string, render: RenderFn) {
+        const string = render(text.trim()).trim();
```

Comment:
would it ever be possible a user has intentionally added a leading or trailing whitespace that they want encoded?

Reply:
hmm ... good question!  

For the other lambdas, spaces aren't important, so we do all the `trim()`'s, but for this one - it's basically a string transform into a different string, and specifically deals with spaces:

```console
$ node
> encodeURI(' ')
'%20'
```

So seems like we should **NOT** do the `trim()`'s.